tests/tests_api/test_group.py

```diff
@@ -77,24 +87,75 @@ def test_public_group_retrieve(api_client):
         "telegram_chat_url": group.telegram_chat_url,
         "coordinators": [
             {
-                "id": coordinator.id,
+                "id": coordinator.pk,
                 "full_name": coordinator.personal_info.full_name,
             }
             for coordinator in group.coordinators.all()
         ],
         "teachers": [
             {
-                "id": teacher.id,
+                "id": teacher.pk,
                 "full_name": teacher.personal_info.full_name,
             }
             for teacher in group.teachers.all()
         ],
         "students": [
             {
-                "id": student.id,
+                "id": student.pk,
                 "full_name": student.personal_info.full_name,
             }
             for student in group.students.all()
         ],
         "is_for_staff_only": group.is_for_staff_only,
     }
+
+
+class TestPublicGroupStart:
+    def test_public_group_start_general_check(self, api_client):
+        group = baker.make(Group, _fill_optional=True, make_m2m=True)
+
+        with transaction.atomic():
+            group.status = GroupStatus.AWAITING_START
+            # to make sure the status_since really gets updated:
+            group.status_since = timezone.now() - datetime.timedelta(days=1, hours=1, minutes=10)
+            group.save()
+            group.coordinators.update(status=CoordinatorStatus.WORKING_BELOW_THRESHOLD)
+            group.students.update(status=StudentStatus.AWAITING_START)
+            group.teachers.update(status=TeacherStatus.AWAITING_START)
+
+        response = api_client.post(self._make_url(group))
+
+        assert response.status_code == status.HTTP_201_CREATED
+
+        group = Group.objects.get(pk=group.id)  # reload, or else status will be the old one
+        assert group.status == GroupStatus.WORKING
+
+        common_status_since = group.status_since
+        assert common_status_since.day == timezone.now().day
+        assert common_status_since.hour == timezone.now().hour
+        assert common_status_since.minute == timezone.now().minute
```

Comment:
timezone.now() called three times, in theory it could lead to some problems. Also for such cases it's better to use freezegun (https://github.com/spulec/freezegun)

Reply:
Not sure we need it right now, but it's definitely better to call timezone.now() only once

Reply:
Done

Reply:
I'll keep `freezegun` in mind, for now I think it's OK if I just test the minutes and the hours (especially because I explicitly set `status_since` to yesterday before running `start`)